### PR TITLE
Adding example for kexec

### PR DESCRIPTION
**Pull request by sounix000**
### PR creator: Description

Adding a simple example for the `kexec` command to make it clear what and how the command can be executed. 

### PR creator: Are there any relevant issues/feature requests?

* [bsc#1194973](https://bugzilla.suse.com/show_bug.cgi?id=1194973)
* [jira#DOCTEAM-504](https://jira.suse.com/browse/DOCTEAM-504)

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [x] SLE 15 SP4/openSUSE Leap 15.5
  - [x] SLE 15 SP4/openSUSE Leap 15.4
  - [x] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [x] all necessary backports are done

### CODE REVIEW

**Review by tbazant — APPROVED** [review]
Looks good to me

File: xml/tuning_kexec.xml [hunk 1]
```diff
@@ -189,6 +193,15 @@
    /proc/cmdline) more_options"</replaceable>.
   </para>
 
+  <para>
+   For example, to load the <filename>/boot/vmlinuz-5.14.21-150500.53-default</filename> kernel image
+   with the command-line of the currently running production kernel and the
```
Comment (taroth21) [inline xml/tuning_kexec.xml:198]:
```suggestion
   with the command line of the currently running production kernel and the
```

File: xml/tuning_kexec.xml [hunk 2]
```diff
@@ -189,6 +193,15 @@
...
+   <filename>/boot/initrd</filename> initrd file, run the following command:
```
Comment (taroth21) [inline xml/tuning_kexec.xml:199]:
```suggestion
   <filename>/boot/initrd</filename> file, run the following command:
```

File: xml/tuning_kexec.xml [hunk 3]
```diff
@@ -189,6 +193,15 @@
...
+  </para>
+<screen>
+&prompt.root; kexec -l /boot/vmlinuz-5.14.21-150500.53-default --append="$(cat /proc/cmdline)" --initrd=/boot/initrd
+</screen>
```
Comment (taroth21) [inline xml/tuning_kexec.xml:203]:
```suggestion
<screen>&prompt.root; kexec -l /boot/vmlinuz-5.14.21-150500.53-default --append="$(cat /proc/cmdline)" --initrd=/boot/initrd</screen>
```

**Review by taroth21 — CHANGES_REQUESTED** [review]
Hi Souvik, I suggested a few changes to follow our style guide rules (spelling of command line as noun and avoiding line breaks within the `<screen/>` element (your formatting would have resulted in an empty line before and after the screen).

File: xml/tuning_kexec.xml [hunk 4]
```diff
@@ -189,6 +193,13 @@
    /proc/cmdline) more_options"</replaceable>.
   </para>
 
+  <para>
+   For example, to load the <filename>/boot/vmlinuz-5.14.21-150500.53-default</filename> kernel image
+   with the command line of the currently running production kernel and the
+   <filename>/boot/initrd</filename> file, run the following command:
+  </para>
+<screen>&prompt.root; kexec -l /boot/vmlinuz-5.14.21-150500.53-default --append="$(cat /proc/cmdline)" --initrd=/boot/initrd</screen>
```
Comment (taroth21) [inline xml/tuning_kexec.xml:201]:
```suggestion
<screen>&prompt.root; kexec -l /boot/vmlinuz-5.14.21-150500.53-default \
--append="$(cat /proc/cmdline)" --initrd=/boot/initrd</screen>
```

**Review by taroth21 — APPROVED** [review]

**Pull request merged**